osu.Game/Screens/Edit/Setup/SetupSection.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
+using osuTK;
+
+namespace osu.Game.Screens.Edit.Setup
+{
+    internal class SetupSection : Container
+    {
+        protected FillFlowContainer Flow;
+
+        [Resolved]
+        protected OsuColour Colours { get; private set; }
+
+        [Resolved]
+        protected IBindable<WorkingBeatmap> Beatmap { get; private set; }
+
+        public override void Add(Drawable drawable) => throw new InvalidOperationException("Use Flow.Add instead");
```

Comment:
Is there a reason why this is not set as override of `Content`?

Reply:
FillFlowContainer ain't a Container :(

Reply:
Oh, you might need to use `FillFlowContainer<Drawable>`. I'm 99% sure this works.

Reply:
Actually no, am I misunderstanding something? It should just work by the looks of it:
![image](https://user-images.githubusercontent.com/1329837/95190078-b6d3e500-0809-11eb-944e-779cca3884c5.png)

Reply:
Weird, it does work. I may have changed the type since I determined it didn't.